MewsSystems/mews-flutter
# feat: [UX-137] Add Chip component

#### Summary

- added `OptimusChip` component (this one is meant to replace the interactive tag)
- added Story

#### Testing steps

1. Check `Chip` story
2. Check if everything is displayed correctly using the exposed controls

#### Follow-up issues

I'll remove the interactive tag variant when I'll be updating the tags to the new design.

#### Check during review

- Verify against Jira issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).
